### ISSUE TITLE
Could you please provide the training log on the ImageNet for reference?

### ISSUE THREAD

**Issue by gongchenting** [sail-sg/poolformer#20]

**Comment by yuweihao** [comment]
Hi @gongchenting , 

Sure. Please see this [Google Drive folder](https://drive.google.com/drive/folders/1uUkYW3klhm-o288LjKrp65_S-wngXRFV?usp=sharing) for the training log (batch size of 4096, learning rate of 4e-3, on TPUs).

Happy new year in advance :)
